components/general/AddressItem.tsx

```diff
@@ -0,0 +1,31 @@
+import styles from "@/styles/AddressItem.module.css";
+import { fontSatoshi, fontRobotoMono, fontSyne } from "@/pages/_app";
```

Comment:
Because you have to much components, I think it is good for read if you put all related component's file in one sub-folder in general folder~

Reply:
Good suggestion